pynxtools_apm/utils/oasis_eln_reader.py

```diff
@@ -59,8 +59,8 @@ class NxApmNomadOasisElnSchemaParser:
     def __init__(self, file_path: str, entry_id: int, verbose: bool = False):
         print(f"Extracting data from ELN file: {file_path}")
         if (
-            file_path.rsplit("/", 1)[-1].startswith("eln_data")
-            or file_path.startswith("eln_data")
+            file_path.rsplit("/", 1)[-1].endswith("eln_data.yaml")
+            or file_path.rsplit("/", 1)[-1].endswith("eln_data.yml")
```

Comment:
Consider using `pathlib.Path(file_path).name`, which is a bit more expressive.
I think you can also allow any filename here and just check for the file ending (`os.path.splitext(file_path)[1] in (".yaml", ".yml")`)

Reply:
implemented, no sticking with eln_data convention, indeed in the future should also use e.g anyname.eln and using the ELN file format